api-catalog-services/src/main/resources/application.yml

```diff
@@ -15,6 +15,7 @@ spring:
         ansi:
             enabled: detect
     main:
+        allow-circular-references: true
```

Comment:
Why is this now needed?

Reply:
This one will have an issue opened as well?

Reply:
#2942